.travis.yml

```
@@ -4,7 +4,6 @@ os:
   - linux
   - osx
 julia:
-  - 0.7
   - 1.0
```

```suggestion
  - 1.0
  - 1.1
```
Since 1.1 was added to AppVeyor